webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -61,6 +61,7 @@ export default class CreateIssueModal extends PureComponent {
         'priority',
         'description',
         'summary',
+        'reporter',
```

Comment:
For clarification of how this change will affect the flow:

The user will now be able to fill in the reporter field in the dialog window.
* If they leave it empty, it will be handled server side and set to themselves.
* If they enter a value `jiraClient.Issue.Create(issue)` will attempt to run and return an error if reporter not found.
  * Err will be displayed at top of dialog window.

Reply:
Hmmm, @jasonblais this may not be what you'd want.

Reply:
@levb UX-wise there should be no change, preferably.

1. We don't handle the ability to select users, therefore someone who tries to select a Reporter won't work. It'll likely fail as @jfrerich reported above.
2. Any UX changes are risky this close to release date.

When the `Reporter` field is optional, we assign the issue creator as the `Reporter`. Can we do the same when the field is required?

Reply:
The short answer is, no. Let me see if I can do this all in the client instead.

Reply:
Cool. I'm also open to discussing other options